rd_ui/app/scripts/directives/dashboard_directives.js

```diff
@@ -123,6 +123,9 @@
         replace: true,
         link: function($scope, element, attrs) {
           $scope.widgetSizes = [{
+            name: 'Hidden',
+            value: 0
+          }, {
```

Comment:
Check [line 151](https://github.com/mmelograno/redash/blob/master/rd_ui/app/scripts/directives/dashboard_directives.js#L151) in this file, we already add the hidden option there.